docs/index.md

```diff
@@ -30,48 +28,30 @@ This means that you can write your entire application in a single language, Rust
+## State Mirroring
 
-Additionally, each database has an owner `Identity`. Many database maintenance operations, like publishing a new version or evaluating arbitrary SQL queries, are restricted to only authenticated connections by the owner.
+SpacetimeDB can generate client code in a [variety of languages](#client-side-sdks). This creates a client library custom-designed to talk to your module. It provides easy-to-use interfaces for logging in and submitting requests. It can also **automatically mirror state** from your module's database.
 
-SpacetimeDB provides tools in the CLI and the [client SDKs](/docs/sdks) for managing credentials.
-
-## Addresses
-
-A SpacetimeDB `Address` is an opaque identifier for a database or a client connection. An `Address` is a 128-bit integer, usually formatted as a 32-character (16-byte) hexadecimal string.
-
-Each SpacetimeDB database has an `Address`, generated by the SpacetimeDB host, which can be used to connect to the database or to request information about it. Databases may also have human-readable names, which are mapped to addresses internally.
-
-Each client connection has an `Address`. These addresses are opaque, and do not correspond to any metadata about the client. They are notably not IP addresses or device identifiers. A client connection can be uniquely identified by its `(Identity, Address)` pair, but client addresses may not be globally unique; it is possible for multiple connections with the same `Address` but different identities to co-exist. SpacetimeDB modules should treat `Identity` as differentiating users, and `Address` as differentiating connections by the same user.
+You write SQL queries specifying what information a client is interested in -- for instance, the terrain and items near a player's avatar. SpacetimeDB's built-in ORM will generate types in your client language for the relevant tables, and feed your client live updates whenever the database state changes. Don't worry about security, this is a **read-only** mirror -- the only way to change the database is to submit requests, which are validated on the server.
 
 ## Language Support
 
 ### Server-side Libraries
 
-Currently, Rust is the best-supported language for writing SpacetimeDB modules. Support for lots of other languages is in the works!
+Currently, Rust is the best-supported language for writing SpacetimeDB modules. C# is also well-supported.
 
 - [Rust](/docs/modules/rust) - [(Quickstart)](/docs/modules/rust/quickstart)
 - [C#](/docs/modules/c-sharp) - [(Quickstart)](/docs/modules/c-sharp/quickstart)
-- Python (Coming soon)
-- Typescript (Coming soon)
+- Python (Planned)
+- Typescript (Planned)
 - C++ (Planned)
 - Lua (Planned)
```

Comment:
Remove these. Our future plans belong on the (forthcoming) roadmap page, and nowhere else in our docs.